hircluster.c

```diff
@@ -3630,7 +3637,10 @@ redisAsyncContext *actx_get_by_node(redisClusterAsyncContext *acc,
         if (ac->c.err == 0) {
             return ac;
         } else {
-            NOT_REACHED();
+            /* Disconnect ongoing and __redisAsyncDisconnect() will eventually */
+            /* call unlinkAsyncContextAndNode() before freeing the context.    */
+            __redisClusterAsyncSetError(acc, ac->c.err, ac->c.errstr);
+            return NULL;
```

Comment:
I don't fully understand the comment. I suppose there is a context but it is invalid or has an error set. (Its corresponding comment in the PR description "a user attempts to send a command while a connection is disconnecting".)

Can you explain it better, possibly split into two sentences?

Reply:
Good, have updated comments with something more understandable I think